core/tests/family_test.cc

```diff
@@ -20,8 +20,8 @@ TEST(FamilyTest, labels) {
                          {{const_label.name, const_label.value}}};
   family.Add({{dynamic_label.name, dynamic_label.value}});
   auto collected = family.Collect();
-  ASSERT_GE(collected.size(), 1);
-  ASSERT_GE(collected.at(0).metric.size(), 1);
+  ASSERT_GE(collected.size(), 1U);
```

Comment:
I think the literal `1` is ok. It's unnecessary to use `1U`.

Reply:
I enabled warnings. size() returns an unsigned type and the warning complains about a compare between signed and unsigned.

Reply:
Ok, I see.